docs/eom_base.rst

```diff
@@ -0,0 +1,372 @@
+==============================
+Two Mass Spring Damper Example
+==============================
+
+This example walks through the creation of a two mass spring damper system,
+manually entered to the `eombase` class and numerically simulated using
+`pydy.system.System`. ::
+
+    >>> import matplotlib.pyplot as plt
+    >>> from numpy import linspace
+    >>> from pydy.system import System
+    >>> from sympy import symbols, Matrix
+    >>> from sympy.physics.mechanics import dynamicsymbols, eombase
```

Comment:
`>>> from sympy.physics.mechanics.eombase import EOM`

Symbolic Equations Of Motion: EoM, EOM, EquationsOfMotion

`*Methods` classes will generate equations of motion.